<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Peoplemessage</class>
 <widget class="QDialog" name="Peoplemessage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>0</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>私人资料</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>252</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>30</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>编辑</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="address">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>地址：</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>20</y>
     <width>421</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>账号：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="zhanghao"/>
    </item>
    <item row="0" column="2" colspan="2">
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>姓名：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLineEdit" name="name"/>
    </item>
    <item row="0" column="6" colspan="2">
     <widget class="QLabel" name="label_14">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>性别：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBox"/>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_12">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3" colspan="2">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>身份证号：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5" colspan="2">
     <widget class="QLineEdit" name="idnum"/>
    </item>
    <item row="1" column="7">
     <widget class="QLabel" name="label_13">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>电话：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="telnum"/>
    </item>
    <item row="2" column="2" colspan="2">
     <widget class="QLabel" name="label_10">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>邮箱：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QLineEdit" name="email"/>
    </item>
    <item row="2" column="6" colspan="2">
     <widget class="QLabel" name="label_11">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pushButton</tabstop>
  <tabstop>zhanghao</tabstop>
  <tabstop>name</tabstop>
  <tabstop>telnum</tabstop>
  <tabstop>email</tabstop>
  <tabstop>address</tabstop>
  <tabstop>idnum</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
